lib/bundler/cli/install.rb

```diff
@@ -114,6 +116,20 @@ def warn_if_root
         "application for all non-root users on this machine.", :wrap => true
     end
 
+    def warn_if_outdated
+      return if ENV["BUNDLE_POSTIT_TRAMPOLINING_VERSION"].nil?
+      installed_version = ENV["BUNDLE_POSTIT_TRAMPOLINING_VERSION"]
+      installed_version = Gem::Version.new(installed_version)
```

Comment:
This was an attempt to fix breaking specs on 1.9.3. I'll be removing the commit that did this in a few hours when I arrive at uni.